_data-prepper/pipelines/configuration/processors/parse-json.md

```diff
@@ -1,20 +1,20 @@
 ---
 layout: default
-title: json
+title: Parse json
 parent: Processors
 grand_parent: Pipelines
 nav_order: 45
 ---
 
-# json
+# Parse json
 
 ## Overview
 
-Takes in an event and parses its JSON data, including any nested fields.
+The `Parse json` processor takes in an event and parses its JSON data, including any nested fields.
```

Comment:
If we use the code formatting, we should use the name from the config. In this case, `parse_json`.

We may want to have a standard convention here.